src/twisted/words/protocols/irc.py

```diff
@@ -1713,15 +1717,18 @@ def _safeMaximumLineLength(self, command):
         fudge = 10
         return MAX_COMMAND_LENGTH - len(theoretical) - fudge
 
-    def msg(self, user, message, length=None):
+    def _send_msg_or_notice(self, msg_or_notice, user, message, length=None):
```

Comment:
By the Twisted naming convention this should be `_sendMsgOrNotice`. The same applies to the `msg_or_notice` argument &mdash; `msgOrNotice` &mdash; though the naming is awkward in both cases. I see the IRC3 spec refers to PRIVMSG and NOTICE as "messages". Perhaps call the function `_sendMessage` and the argument `command`?

Reply:
I wanted to avoid ambiguous names, but if the specs refer to both as "messages", it should be allright.